assests/js/index.js

```diff
@@ -0,0 +1,217 @@
+//Variable declaration
+const dateText = $("#currentDay");
+const timeText = $("#currentTime");
+const timeBlock = $(".container");
+const currentHr = moment().hour();
+
+const scheduleKey = "schedules";
+
+// use moment to get current date & time in format & real time update
+const getCurrentDate = function () {
+  const timer = function () {
+    const date_Time = moment();
+    const dateTime_Formatted = date_Time.format(
+      "dddd, MMMM Do YYYY, hh:mm:ss a "
+    );
+
+    dateText.text(dateTime_Formatted);
+  };
+  timer();
+  setInterval(timer, 1000);
+};
+
+//Save data to LS
+const saveSchedule = function (event) {
+  const eventTarget = $(event.target);
+  const attr = eventTarget.attr("data-key");
+  const textarea = $(`#${attr}`);
+
+  if (eventTarget.is("#saveBtn")) {
+    const textValue = textarea.val();
+    let data = retrieveData(scheduleKey);
+
+    const schedule = { key: attr, appointment: textValue };
+    const text = getItem(data, attr);
+
+    if (text) {
+      data.map(function (item, index) {
+        if (item.key == attr.toString()) {
+          data[index] = schedule;
+        }
+        return data;
+      });
+    } else {
+      data.push(schedule);
+    }
+
+    savesData(scheduleKey, data);
+  }
+};
+
+const getItem = function (data, key) {
+  const itemValue = data.find(function (item) {
+    if (item.key == key.toString()) {
+      return item;
+    }
+  });
+
+  return itemValue;
+};
+const savesData = function (dataKey, data) {
+  localStorage.setItem(dataKey, JSON.stringify(data));
+};
+
+const retrieveData = function (dataKey) {
+  const data = JSON.parse(localStorage.getItem(dataKey)) || [];
+
+  if (data.length > 0) {
```

Comment:
Here, you could just return `data` instead of doing an `if` statement to get the same result. Above, you have declared `data` as the data from local storage or []. Therefore, if there was no data in local storage, then `data` is already set to [] and this will be returned.

Reply:
updated with only return data
thank you